fourcc.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import sys
+
+cc = os.getenv("CC", "cc")
+pkg_config = os.getenv("PKG_CONFIG", "pkg-config")
+
+cflags = subprocess.check_output([pkg_config, "libdrm", "--cflags-only-I"]).decode().strip().split()
```

Comment:
Getting the directory from Meson and passing it as a parameter to the script is probably better

Reply:
Got it.